routers/web/goget.go

```diff
@@ -67,7 +67,7 @@ func goGet(ctx *context.Context) {
 	}
 	ctx.RespHeader().Set("Content-Type", "text/html")
 	ctx.Status(http.StatusOK)
-	_, _ = ctx.Write([]byte(com.Expand(`<!doctype html>
+	res, err := vars.Expand(`<!doctype html>
```

Comment:
```suggestion
	goGetImport := context.ComposeGoGetImport(ownerName, trimmedRepoName)
	goDocDirectory := prefix + "{/dir}"
	ctx.Write([]byte(fmt.Sprintf(`<!doctype html>
<html>
	<head>
		<meta name="go-import" content="%s git %s">
		<meta name="go-source" content="%s _ %s %s">
	</head>
	<body>
		go get %s
	</body>
</html>
`, goGetImport, repo_model.ComposeHTTPSCloneURL(ownerName, repoName), goGetImport, goDocDirectory, goDocDirectory + "/{file}#L{line}", insecure+goGetImport)))
```
(but it's probably better if you copy this suggestion into your IDE and manually format it, as it's not unlikely that I made a formatting error, and also I can't throw out the lines below for some reason...)

Reply:
I use `os.Expand` to replace `vars.Expand` here.